lib/white_bread/context.ex

```diff
@@ -110,5 +113,15 @@ defmodule WhiteBread.Context do
     end
   end
 
-
+  defp import_test_library(test_library) do
```

Comment:
Yes, this is a function that returns a quoted block. It seemed like the right thing here, but I'm open to other ideas (considering I'm a relative beginner in Elixir...).

Reply:
Seems a sensible approach to me here. It's relatively easy to follow what it's doing.

Reply:
I realized that I don't know the answer to this question: if macros are evaluated at compile time, will the fact that this is a function cause any problems? (It's been working well for me, but I just want to make sure that I didn't stumble into an Elixir gotcha.)